.github/workflows/post-merge.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Parse Node version
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        run: echo NVMRC=`cat .nvmrc` >> $GITHUB_OUTPUT
```

Comment:
For consistency sake can we make this line same as [this line](https://github.com/alphagov/pay-selfservice/pull/3791/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R29)? So 
```
echo "NVMRC=$(cat .nvmrc)" >> $GITHUB_OUTPUT
```
This is also the format suggested in https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.

Reply:
I tried that format as suggested prior to this in another repo https://github.com/alphagov/pay-products-ui/pull/2317 which caused the tests to fail, hence had to figure out a different way. This is not failing the test.

Reply:
Hmm I'm not convinced as per the reasons I gave before:
* you're using [this format in this same PR](https://github.com/alphagov/pay-selfservice/pull/3791/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R29)
* `echo "{name}={value}" >> $GITHUB_OUTPUT` is the format recommended by github - https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
I agree with Oz - the curly braces format should work here (if it doesn't there's probably something else wrong). Using backticks for evaluating a statement instead of `${ ... }` is not recommended (the backticks are deprecated in bash).